### PR TITLE
Feat/kubernetes

### PR DESCRIPTION
**Pull request by marie-j**
# Migrate Kubernetes 

Add `@ovh-ux/manager-kubernetes` package 

## Description of the change

git-subtree-dir: packages/manager/modules/kubernetes/src
git-subtree-split: d25b04a0185cafcb669bad1897537c8fc052ac4d

**ref: OM-297**

⚠️ Commits might need to be squashed 

## TODOS 

- [ ] Use in `ovh-manager-cloud` 
- [x] Split `KUBERNETES` constant

/cc @jleveugle @frenautvh @antleblanc @cbourgois

### CODE REVIEW

**Comment by ovh-cds** [comment]
CDS Report installation#1242.0 ■   
* Stage 1
  * Install ■

**Comment by ovh-cds** [comment]
CDS Report installation#1242.0 ■   
* Stage 1
  * Install ■

File: packages/manager/modules/kubernetes/package.json [hunk 1]
```diff
@@ -0,0 +1,47 @@
+{
+  "name": "@ovh-ux/manager-kubernetes",
+  "description": "OVH Kubernetes product control panel",
+  "version": "0.0.0",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/ovh-ux/manager.git",
+    "directory": "packages/manager/modules/kubernetes"
+  },
+  "license": "BSD-3-Clause",
```
Comment (frenautvh) [inline packages/manager/modules/kubernetes/package.json]:
```suggestion
  "license": "BSD-3-Clause",
  "files": [
    "dist"
  ],
```

File: packages/manager/modules/kubernetes/package.json [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  "peerDependencies": {
+    "@ovh-ux/manager-cloud-styles": "^0.2.0-alpha.0",
+    "@ovh-ux/manager-core": "^4.0.0",
+    "@ovh-ux/ng-ovh-cloud-universe-components": "^1.0.1-alpha.0",
+    "@ovh-ux/ng-ovh-sidebar-menu": "^8.0.0-beta.1",
+    "@ovh-ux/ng-translate-async-loader": "^2.0.0",
+    "@uirouter/angularjs": "^1.0.22",
+    "angular": "^1.7.8",
+    "angular-ui-bootstrap": "^1.3.3",
+    "bootstrap4": "twbs/bootstrap#v4.0.0",
```
Comment (frenautvh) [inline packages/manager/modules/kubernetes/package.json]: i think it's a dev dependency

File: packages/manager/modules/kubernetes/src/component.js [hunk 3]
```diff
@@ -0,0 +1,54 @@
+import angular from 'angular';
+
+// import 'script-loader!bootstrap/dist/js/bootstrap.js'; // eslint-disable-line
```
Comment (cbourgois) [inline packages/manager/modules/kubernetes/src/component.js]: Should be removed.

**Comment by ovh-cds** [comment]
CDS Report installation#1274.0 ✘  
* Stage 1
  * Install ✘

**Pull request merged**